Description: Replace relative location by hardcoded
Forwarded: no
Author: Ivan Kohler <ivan-debian@420.am>
Reviewed-By: Xavier Guimard <x.guimard@free.fr>,
 gregor herrmann <gregoa@debian.org>
Last-Update: 2013-02-17

--- a/bin/crunchjpgs.pl
+++ b/bin/crunchjpgs.pl
@@ -13,7 +13,7 @@
 
 my %opts = (
             # Hardcoded:
-            template   => 'crunchjpg_tmpl.pdf',
+            template   => '/usr/share/libcam-pdf-perl/crunchjpg_tmpl.pdf',
 
             # User settable values:
             justjpgs   => 0,
--- a/bin/extractallimages.pl
+++ b/bin/extractallimages.pl
@@ -11,7 +11,7 @@
 our $VERSION = '1.60';
 
 my %opts = (
-            template   => 'crunchjpg_tmpl.pdf',
+            template   => '/usr/share/libcam-pdf-perl/crunchjpg_tmpl.pdf',
 
             verbose    => 0,
             help       => 0,
--- a/bin/extractjpgs.pl
+++ b/bin/extractjpgs.pl
@@ -11,7 +11,7 @@
 our $VERSION = '1.60';
 
 my %opts = (
-            template   => 'crunchjpg_tmpl.pdf',
+            template   => '/usr/share/libcam-pdf-perl/crunchjpg_tmpl.pdf',
 
             verbose    => 0,
             help       => 0,
